cost-analyzer/README.md

```diff
@@ -46,3 +46,6 @@ Parameter | Description | Default
 `serviceAccount.create` | Set this to `false` if you want to create the service account `kubecost-cost-analyzer` on your own | `true`
 `tolerations` | node taints to tolerate | `[]`
 `affinity` | pod affinity | `{}`
+`kubecostFrontend.api.fqdn` | Customize the upstream api FQDN | `kubecost-api.kubecost.svc.cluster.local:9001`
+`kubecostFrontend.model.fqdn` | Customize the upstream model FQDN | `kubecost-model.kubecost.svc.cluster.local:9003`
+`clusterController.fqdn` | Customize the upstream cluster controller FQDN | `kubecost-cluster-controller.kubecost.svc.cluster.local:9731`
```

Comment:
Actually default values is different right now.
For example, as I see default value for `kubecostFrontend.api.fqdn` equal to empty string.
And in this case upstream host is `{{ $serviceName }}.{{ .Release.Namespace }}:9001;` and not a `kubecost-api.kubecost.svc.cluster.local:9001`

Reply:
Thanks for spotting that @evgenii-denisov! I've removed those examples as they were not appropriate.